lib/src/main/java/io/ably/lib/realtime/ChannelBase.java

```diff
@@ -91,6 +93,17 @@ public abstract class ChannelBase extends EventEmitter<ChannelEvent, ChannelStat
      */
     private boolean released = false;
 
+    private final LiveObjectsPlugin liveObjectsPlugin;
+
+    public LiveObjects getObjects() throws AblyException {
+        if (liveObjectsPlugin == null) {
+            throw AblyException.fromErrorInfo(
+                new ErrorInfo("LiveObjects plugin hasn't been installed, add runtimeOnly('io.ably:live-objects:<ably-version>') to your dependency tree", 400, 40000)
```

Comment:
I think t good to have specific code error for this, also it's probably worth to point to the liveobject docs in the error

Reply:
will check wrt spec/ js code